### ISSUE TITLE
`RemoveUnusedDeclaration` quick fix silently fails

### ISSUE THREAD

**Issue by daFreeMan** [rubberduck-vba/Rubberduck#5299]
**Rubberduck version information**
```
Version 2.4.1.5200
OS: Microsoft Windows NT 10.0.17134.0, x64
Host Product: Microsoft Office 2016 x64
Host Version: 16.0.4873.1000
Host Executable: MSACCESS.EXE
```

**Description**
Executing the `Remove unused declaration` quick fix is logging errors, but fails silently to the user.

**To Reproduce**
1. Write a procedure
2. Don't call it
3. Parse
3. Execute the `Removed unused procedure` quick fix on it.

**Expected behavior**
The unused procedure should have been removed from code.

**Logfile**
[RubberduckLog.txt](https://github.com/rubberduck-vba/Rubberduck/files/3880056/RubberduckLog.txt)

**Additional context**
The two errors thrown:
```
2019-11-22 09:38:57.3383;ERROR-2.4.1.5200;Rubberduck.UI.Inspections.InspectionResultsViewModel;System.Runtime.InteropServices.COMException (0x80020010): Invalid callee. (Exception from HRESULT: 0x80020010 (DISP_E_BADCALLEE))
   at Microsoft.Vbe.Interop._CodePane.get_CodeModule()
   at Rubberduck.VBEditor.SafeComWrappers.VBA.CodePane.get_CodeModule() in C:\projects\rubberduck\Rubberduck.VBEditor.VBA\SafeComWrappers\VB\CodePane.cs:line 28
   at Rubberduck.VBEditor.SafeComWrappers.VBA.CodePane.get_QualifiedModuleName() in C:\projects\rubberduck\Rubberduck.VBEditor.VBA\SafeComWrappers\VB\CodePane.cs:line 77
   at Rubberduck.VBEditor.Utility.SelectionService.OpenModules() in C:\projects\rubberduck\Rubberduck.VBEEditor\Utility\SelectionService.cs:line 45
   at Rubberduck.Parsing.Rewriter.SelectionRecoverer.SaveSelections(IEnumerable`1 modules) in C:\projects\rubberduck\Rubberduck.Parsing\Rewriter\SelectionRecoverer.cs:line 31
   at Rubberduck.Parsing.Rewriter.RewriteSessionBase.PrimeSelectionRecovery() in C:\projects\rubberduck\Rubberduck.Parsing\Rewriter\RewriteSessionBase.cs:line 108
   at Rubberduck.Parsing.Rewriter.RewriteSessionBase.TryRewrite() in C:\projects\rubberduck\Rubberduck.Parsing\Rewriter\RewriteSessionBase.cs:line 99
   at Rubberduck.Inspections.QuickFixes.QuickFixProvider.Fix(IQuickFix fix, IInspectionResult result) in C:\projects\rubberduck\Rubberduck.CodeAnalysis\QuickFixes\QuickFixProvider.cs:line 95
   at Rubberduck.UI.Command.CommandBase.Execute(Object parameter) in C:\projects\rubberduck\Rubberduck.Core\UI\Command\CommandBase.cs:line 85;System.Runtime.InteropServices.COMException (0x80020010): Invalid callee. (Exception from HRESULT: 0x80020010 (DISP_E_BADCALLEE))
   at Microsoft.Vbe.Interop._CodePane.get_CodeModule()
   at Rubberduck.VBEditor.SafeComWrappers.VBA.CodePane.get_CodeModule() in C:\projects\rubberduck\Rubberduck.VBEditor.VBA\SafeComWrappers\VB\CodePane.cs:line 28
   at Rubberduck.VBEditor.SafeComWrappers.VBA.CodePane.get_QualifiedModuleName() in C:\projects\rubberduck\Rubberduck.VBEditor.VBA\SafeComWrappers\VB\CodePane.cs:line 77
   at Rubberduck.VBEditor.Utility.SelectionService.OpenModules() in C:\projects\rubberduck\Rubberduck.VBEEditor\Utility\SelectionService.cs:line 45
   at Rubberduck.Parsing.Rewriter.SelectionRecoverer.SaveSelections(IEnumerable`1 modules) in C:\projects\rubberduck\Rubberduck.Parsing\Rewriter\SelectionRecoverer.cs:line 31
   at Rubberduck.Parsing.Rewriter.RewriteSessionBase.PrimeSelectionRecovery() in C:\projects\rubberduck\Rubberduck.Parsing\Rewriter\RewriteSessionBase.cs:line 108
   at Rubberduck.Parsing.Rewriter.RewriteSessionBase.TryRewrite() in C:\projects\rubberduck\Rubberduck.Parsing\Rewriter\RewriteSessionBase.cs:line 99
   at Rubberduck.Inspections.QuickFixes.QuickFixProvider.Fix(IQuickFix fix, IInspectionResult result) in C:\projects\rubberduck\Rubberduck.CodeAnalysis\QuickFixes\QuickFixProvider.cs:line 95
   at Rubberduck.UI.Command.CommandBase.Execute(Object parameter) in C:\projects\rubberduck\Rubberduck.Core\UI\Command\CommandBase.cs:line 85
2019-11-22 09:39:07.0492;ERROR-2.4.1.5200;Rubberduck.UI.Inspections.InspectionResultsViewModel;System.Runtime.InteropServices.COMException (0x80020010): Invalid callee. (Exception from HRESULT: 0x80020010 (DISP_E_BADCALLEE))
   at Microsoft.Vbe.Interop._CodePane.get_CodeModule()
   at Rubberduck.VBEditor.SafeComWrappers.VBA.CodePane.get_CodeModule() in C:\projects\rubberduck\Rubberduck.VBEditor.VBA\SafeComWrappers\VB\CodePane.cs:line 28
   at Rubberduck.VBEditor.SafeComWrappers.VBA.CodePane.get_QualifiedModuleName() in C:\projects\rubberduck\Rubberduck.VBEditor.VBA\SafeComWrappers\VB\CodePane.cs:line 77
   at Rubberduck.VBEditor.Utility.SelectionService.OpenModules() in C:\projects\rubberduck\Rubberduck.VBEEditor\Utility\SelectionService.cs:line 45
   at Rubberduck.Parsing.Rewriter.SelectionRecoverer.SaveSelections(IEnumerable`1 modules) in C:\projects\rubberduck\Rubberduck.Parsing\Rewriter\SelectionRecoverer.cs:line 31
   at Rubberduck.Parsing.Rewriter.RewriteSessionBase.PrimeSelectionRecovery() in C:\projects\rubberduck\Rubberduck.Parsing\Rewriter\RewriteSessionBase.cs:line 108
   at Rubberduck.Parsing.Rewriter.RewriteSessionBase.TryRewrite() in C:\projects\rubberduck\Rubberduck.Parsing\Rewriter\RewriteSessionBase.cs:line 99
   at Rubberduck.Inspections.QuickFixes.QuickFixProvider.Fix(IQuickFix fix, IInspectionResult result) in C:\projects\rubberduck\Rubberduck.CodeAnalysis\QuickFixes\QuickFixProvider.cs:line 95
   at Rubberduck.UI.Command.CommandBase.Execute(Object parameter) in C:\projects\rubberduck\Rubberduck.Core\UI\Command\CommandBase.cs:line 85;System.Runtime.InteropServices.COMException (0x80020010): Invalid callee. (Exception from HRESULT: 0x80020010 (DISP_E_BADCALLEE))
   at Microsoft.Vbe.Interop._CodePane.get_CodeModule()
   at Rubberduck.VBEditor.SafeComWrappers.VBA.CodePane.get_CodeModule() in C:\projects\rubberduck\Rubberduck.VBEditor.VBA\SafeComWrappers\VB\CodePane.cs:line 28
   at Rubberduck.VBEditor.SafeComWrappers.VBA.CodePane.get_QualifiedModuleName() in C:\projects\rubberduck\Rubberduck.VBEditor.VBA\SafeComWrappers\VB\CodePane.cs:line 77
   at Rubberduck.VBEditor.Utility.SelectionService.OpenModules() in C:\projects\rubberduck\Rubberduck.VBEEditor\Utility\SelectionService.cs:line 45
   at Rubberduck.Parsing.Rewriter.SelectionRecoverer.SaveSelections(IEnumerable`1 modules) in C:\projects\rubberduck\Rubberduck.Parsing\Rewriter\SelectionRecoverer.cs:line 31
   at Rubberduck.Parsing.Rewriter.RewriteSessionBase.PrimeSelectionRecovery() in C:\projects\rubberduck\Rubberduck.Parsing\Rewriter\RewriteSessionBase.cs:line 108
   at Rubberduck.Parsing.Rewriter.RewriteSessionBase.TryRewrite() in C:\projects\rubberduck\Rubberduck.Parsing\Rewriter\RewriteSessionBase.cs:line 99
   at Rubberduck.Inspections.QuickFixes.QuickFixProvider.Fix(IQuickFix fix, IInspectionResult result) in C:\projects\rubberduck\Rubberduck.CodeAnalysis\QuickFixes\QuickFixProvider.cs:line 95
   at Rubberduck.UI.Command.CommandBase.Execute(Object parameter) in C:\projects\rubberduck\Rubberduck.Core\UI\Command\CommandBase.cs:line 85
```

**Comment by MDoerner** [comment]
This seems to be some COM interop problem rearing its head again. Based on the documentation of the object model, this call should never fail.

**Comment by daFreeMan** [comment]
Hrm... just for giggles, I saved my .accdb and closed Access. After reopening the file, I did a C&R, parsed, and the duck was able to successfully execute the quick fix.

My .accdb file was ~7MB after the C&R. It's about 14MB now. It was >50Mb before the C&R. Don't know if that had anything to do with it. I've been doing a _lot_ of code cleanup this morning, so I think Access leaves random junk behind from that and requires a bit of tidying up every now and then.

Here's the log from the successful run. 
[RubberduckLog.txt](https://github.com/rubberduck-vba/Rubberduck/files/3880110/RubberduckLog.txt)

This may end up being rather difficult to get a repo on.
